LambdaEngine/Include/Rendering/Core/API/GraphicsTypes.h

```diff
@@ -27,7 +27,7 @@ namespace LambdaEngine
 	constexpr const uint32 BACK_BUFFER_COUNT			= 3;
 
 	// Determines if a resource should be allocated by a deviceallocator or via a seperate allocation
-	constexpr const uint32 LARGE_TEXTURE_ALLOCATION_SIZE				= MEGA_BYTE(64);
+	constexpr const uint32 LARGE_TEXTURE_ALLOCATION_SIZE				= MEGA_BYTE(128);
```

Comment:
Now when the bug is fixed I think that we should use 64 again
```suggestion
	constexpr const uint32 LARGE_TEXTURE_ALLOCATION_SIZE				= MEGA_BYTE(64);
```